src/envs/behavior.py

```diff
@@ -89,7 +89,7 @@ def __init__(self) -> None:
         option_elems = [
             ("NavigateTo", planner_fns[0], option_policy_fns[0],
              option_model_fns[0], 2, 1, (-5.0, 5.0)),
-            ("Grasp", planner_fns[1], option_policy_fns[2],
+            ("Grasp", planner_fns[1], option_policy_fns[1],
```

Comment:
lol oh gosh, was that a bug?

Reply:
yup - it was executing place instead of grasp and I was confused for about an hour...